api_client/python/timesketch_api_client/sketch.py

```diff
@@ -940,8 +940,13 @@ def upload(self, timeline_name, file_path, index=None):
 
         resource_url = f'{self.api.api_root}/upload/'
         files = {'file': open(file_path, 'rb')}
-        data = {'name': timeline_name, 'sketch_id': self.id,
-                'index_name': index}
+        _, _, file_ending = file_path.rpartition('.')
+        data = {
+            'name': timeline_name,
+            'sketch_id': self.id,
+            'label': file_ending,
```

Comment:
Is this the data_source label? Like plaso/csv?

Reply:
data_label